src/Microsoft.Data/DataFrameBuffer.cs

```diff
@@ -44,6 +44,14 @@ public DataFrameBuffer(int numberOfValues = 8)
             Memory = new byte[numberOfValues * _size];
         }
 
+        public DataFrameBuffer(Memory<byte> memory, int length)
+        {
+            _size = Unsafe.SizeOf<T>();
+            //Debug.Assert(length * _size == memory.Length); // Wont work for the nullBitMapBuffer
```

Comment:
Should this just be deleted then?

Reply:
Yup yup, when this patch is really ready to be merged, I'll delete this